content/en/blog/2024/gateway-mesh-ga/index.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "Gateway API Mesh Support Promoted To Stable"
+description: The next-generation Kubernetes traffic routing APIs are now Generally Available for service mesh use cases. 
+publishdate: 2024-05-13
+attribution: John Howard - solo.io
+keywords: [istio, traffic, API]
+target_release: 1.22
+---
+
+We are thrilled to announce that Service Mesh support in the [Gateway API](https://gateway-api.sigs.k8s.io/) is now officially "Stable"!
+With this release (part of Gateway API v1.1 and Istio v1.22), users can make use of the next-generation traffic management APIs for both ingress ("north-south") and service mesh use cases ("east-west").
+
+## What is the Gateway API?
+
+The Gateway API is a collection of APIs that are part of Kubernetes, focusing on traffic routing and management.
+The APIs are inspired by, and serve many of the same roles as, Kubernetes' `Ingress` and Istio's `VirtualService` and `Gateway` APIs.
+
+These APIs have been under development both in Istio, as well as with [broad collaboration](https://gateway-api.sigs.k8s.io/implementations/), since 2020, and have come a long way since then.
+While the API initially targeted only serving ingress use cases (which went GA [last year](https://kubernetes.io/blog/2023/10/31/gateway-api-ga/)), we had always envisioned allowing the same APIs to be used for traffic *within* a cluster as well.
+
+With this release, that vision is made a reality: Istio users can use the same routing API for all of their traffic!
+
+## Getting started
+
+Throughout the Istio documentation, all of our examples have been updated to show how to use the Gateway API, so explore some of the [tasks](/docs/tasks/traffic-management/) to gain a deeper understanding.
+
+Using Gateway API for service mesh should feel familiar both to users already using Gateway API for ingress, and users using `VirtualService` for service mesh today.
+
+* Compared to Gateway API for ingress, routes target a `Service` instead of a `Gateway`.
+* Compared to `VirtualService`, where routes associate with a set of `hosts`, routes target a `Service`.
```

Comment:
Maybe at a warning here about Cluster IP vs. host/authority header routing?

Reply:
Oh wait sorry I'm thinking about Ambient; yeah the warning about not implementing GAMMA all the way is good too